assets/elasticsearch/Dockerfile

```
@@ -0,0 +1,24 @@
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    curl \
+    gpg;
+
+RUN curl -fsSL https://artifacts.elastic.co/GPG-KEY-elasticsearch \
+    | gpg --dearmor -o /usr/share/keyrings/elasticsearch-keyring.gpg \
+    && echo "deb [signed-by=/usr/share/keyrings/elasticsearch-keyring.gpg]" \
+    "https://artifacts.elastic.co/packages/8.x/apt stable main" \
+    | tee /etc/apt/sources.list.d/elastic-8.x.list;
+
+RUN apt-get update \
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)